Build broken after merges

**Problem description**
Build broke after merges #1031 followed by #1095

```
:controller:server:compileTestJava/home/travis/build/pravega/pravega/controller/server/src/test/java/io
/pravega/controller/server/eventProcessor/ControllerEventProcessorTest.java:92: error: no suitable 
method found for process(CommitEvent)
        commitEventProcessor.process(new CommitEvent(SCOPE, STREAM, txnData.getId()));
                            ^
    method EventProcessor.process(CommitEvent,Position) is not applicable
      (actual and formal argument lists differ in length)
    method CommitEventProcessor.process(CommitEvent,Position) is not applicable
      (actual and formal argument lists differ in length)
/home/travis/build/pravega/pravega/controller/server/src/test/java/io/pravega/controller/server/eventPr
ocessor/ControllerEventProcessorTest.java:108: error: no suitable method found for process(AbortEvent)
        abortEventProcessor.process(new AbortEvent(SCOPE, STREAM, txnData.getId()));
                           ^
    method EventProcessor.process(AbortEvent,Position) is not applicable
      (actual and formal argument lists differ in length)
    method AbortEventProcessor.process(AbortEvent,Position) is not applicable
      (actual and formal argument lists differ in length)
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors
 FAILED
```

**Problem location**
Controller tests.

--- a/controller/server/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorTest.java
+++ b/controller/server/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorTest.java
@@ -89,7 +89,7 @@ public class ControllerEventProcessorTest {
 
         CommitEventProcessor commitEventProcessor = new CommitEventProcessor(streamStore, hostStore, executor,
                 segmentHelperMock, null);
-        commitEventProcessor.process(new CommitEvent(SCOPE, STREAM, txnData.getId()));
+        commitEventProcessor.process(new CommitEvent(SCOPE, STREAM, txnData.getId()), null);
         checkTransactionState(SCOPE, STREAM, txnData.getId(), TxnStatus.COMMITTED);
     }
 
@@ -105,7 +105,7 @@ public class ControllerEventProcessorTest {
 
         AbortEventProcessor abortEventProcessor = new AbortEventProcessor(streamStore, hostStore, executor,
                 segmentHelperMock, null);
-        abortEventProcessor.process(new AbortEvent(SCOPE, STREAM, txnData.getId()));
+        abortEventProcessor.process(new AbortEvent(SCOPE, STREAM, txnData.getId()), null);
         checkTransactionState(SCOPE, STREAM, txnData.getId(), TxnStatus.ABORTED);
     }
 

